tests/e2e/testsuites/format_options_tester.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	"fmt"
+	awscloud "github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud"
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/tests/e2e/driver"
+	. "github.com/onsi/ginkgo/v2"
+	v1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// FormatOptionTest will provision required StorageClass(es), PVC(s) and Pod(s) in order to test that volumes with
+// a specified custom format options will mount and are able to be later resized.
+type FormatOptionTest struct {
+	CreateVolumeParameterKey   string
+	CreateVolumeParameterValue string
+}
+
+const (
+	volumeSizeIncreaseAmtGi = 1
+	volumeMountPath         = "/mnt/test-format-option" // TODO should I keep this as mnt/test-1, and refactor to be `DefaultMountPath` globally in testsuites?
```

Comment:
No need to have a custom mount path here, for consistency `mnt/test-1` and thinking about a default mount path variable which is very commonly used throughout the test suite seem like a good idea.